src/main/java/edu/ucsb/cs156/organic/config/SecurityConfig.java

```diff
@@ -55,25 +78,25 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
   // @Autowired
   // private LoginSuccessHandler loginSuccessHandler;
```

Comment:
Please remove the commented out code (yes, I know it was in the original, but the rule is, if you touch the file, you have to clean it up.)